cypress/tests/functional/Classic.spec.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file cypress/tests/functional/Classic.spec.js
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ */
+
+describe('Theme plugin tests', function() {
+	const journalPath = 'publicknowledge';
+	const index = 'index.php';
+	const path = '/' + index + '/' + journalPath;
+	const pages = [
+		'issue/current',
+		'issue/archive',
+		'issue/view/1',
+		'article/view/17',
+		'article/view/17/3',
+		'about',
+		'about/editorialTeam',
+		'about/submissions',
+		'about/contact',
+		'about/privacy',
+		'information/readers',
+		'information/authors',
+		'information/librarians'
+	];
+	const [month, day, year] = new Date()
+		.toLocaleDateString('en-US', {
+			month: 'numeric',
+			day: 'numeric',
+			year: 'numeric'
+		})
+		.split('/');
+	const user = {
+		'givenName': 'John',
+		'familyName': 'Debreenik',
+		'username': 'jdebreenik',
+		'country': 'UA',
+		'affiliation': 'Lorem Ipsum University'
+	};
+
+	it('Enables and selects the theme', function() {
+		cy.login('admin', 'admin', 'publicknowledge');
+
+		cy.get('nav[class="app__nav"] ul a[class="app__navItem"]:contains("Website")').click();
```

Comment:
You'll save yourself some maintenance headaches by reducing the specificity of this selector:

```js
cy.get('.app__nav a:contains("Website")').click();
```